docs/advanced_topics/customisation/admin_templates.md

```diff
@@ -182,7 +182,7 @@ To add extra fields to the login form, override the `fields` block. You will nee
 
 ### `submit_buttons`
 
-To add extra buttons to the login form, override the `submit_buttons` block. You will need to add `{{ block.super }}` somewhere in your block to include the sign in button:
+To add extra buttons to the login form, override the `submit_buttons` block. You will need to add `{{ block.super }}` somewhere in your block to include the sign-in button:
```

Comment:
Great catch 👍 

I had to look it up, but the hyphenated spelling is correct here because sign-in is used as an adjective. 

Today I learned!

Reply:
Thank you!

I am still going through the documentation and I can spot more issues. Should I go ahead and fix them, or just let other contributors work on them?

Reply:
Feel free to fix similar issues 👍